src/native/libs/System.Security.Cryptography.Native/pal_x509.c

```diff
@@ -350,10 +350,17 @@ int32_t CryptoNative_X509StoreCtxRebuildChain(X509_STORE_CTX* ctx)
     return X509_verify_cert(ctx);
 }
 
-void CryptoNative_X509StoreCtxSetVerifyCallback(X509_STORE_CTX* ctx, X509StoreVerifyCallback callback)
+int32_t CryptoNative_X509StoreCtxSetVerifyCallback(X509_STORE_CTX* ctx, X509StoreVerifyCallback callback, void* appData)
 {
     // Just a field mutator, no error queue interactions apply.
```

Comment:
Since this is now capable of failing (and thus throwing):

```suggestion
    ERR_clear_error();
    
```